packages/tasks/scripts/check-snippets.ts

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
I've implemented tests in https://github.com/huggingface/huggingface.js/pull/1003/files using the vite test convention of `xyz.spec.ts` files.

I think we should just put more tests into `xyz.spec.ts` files rather than creating a custom mechanism of `check-snippet.ts` & `inference-check-snippets.yml`

Wdyt?

Reply:
much better yes!

Reply:
I went with this solution because I was bored about how difficult it was to debug things locally but automated tests feel much more natural now that you mention it ^^